gluten-core/src/main/java/org/apache/gluten/substrait/rel/LocalFilesNode.java

```diff
@@ -78,6 +80,7 @@ public enum ReadFileFormat {
     this.partitionColumns.addAll(partitionColumns);
     this.metadataColumns.addAll(metadataColumns);
     this.preferredLocations.addAll(preferredLocations);
+    this.fileReadProperties= properties;
```

Comment:
Do we use this variable in `LocalFilesNode`?

Reply:
Yes, in my situation, this member would be used as the `tableParameters` of `velox::connector::hive::HiveTableHandle`.

Reply:
I see, previously we were setting the fileReadProperties through the setFileReadProperties method. If setFileReadProperties is no longer in use, we can go ahead and remove it.